build.zig

```diff
@@ -15,4 +15,18 @@ pub fn build(b: *std.Build) void {
     const run_cmd = b.addRunArtifact(exe);
     const run_step = b.step("run", "Run the app");
     run_step.dependOn(&run_cmd.step);
+
+    const test_step = b.step("test", "Run library tests");
+    test_step.dependOn(&testStep(b, optimize, target).step);
+}
+
+pub fn testStep(b: *std.Build, optimize: std.builtin.OptimizeMode, target: std.zig.CrossTarget) *std.build.RunStep {
```

Comment:
I'm ultimately trying to eliminate `testStep` as a thing everywhere, so fi you could inline this code in the future that'd be great!